src/components/Popups/ReplyField/ReplyForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from 'box-ui-elements/es/components/button';
+import PrimaryButton from 'box-ui-elements/es/components/primary-button';
+import { EditorState } from 'draft-js';
+import { FormikProps } from 'formik';
+import isEmpty from 'lodash/isEmpty';
+import { FormattedMessage, useIntl } from 'react-intl';
+import ReplyField from './ReplyFieldContainer';
+import { FormValues, Props as OtherProps } from './ReplyFormContainer';
+import messages from '../messages';
+
+type Props = OtherProps & FormikProps<FormValues>;
+
+const ReplyForm = ({
```

Comment:
The directory structure could use with a second look. Should this live in a separate directory from `ReplyField`? Should these components live in `Popups` at all?

Reply:
Hmm good point, I think I'll break out ReplyField and ReplyForm into their own folders